freedomofpress/securedrop
# Git LFS-a-size all images

## Status

Work in Progress

## Description of Changes

Fixes #3176 

Changes proposed in this pull request:

* adds all image formats in docs to tracking in git-lfs
* updates documentation informing doc editors that they need to install git-lfs
* creates a requirements file for ingestion by RTD



## Testing

* Install git-lfs
* Checkout to this branch 
* `git lfs ls-files` to see LFS-enabled files
* `git lfs pull` make sure you have all the things..

Normally there will be no manual manipulation of git lfs commands after this branch gets merged in. You just use git as normal as long as its enabled in your config via `git lfs install`


## Deployment

~There is a potential weird deployment story here with RTD. Specifically in order for RTD to build this branch it will need to install a `git-lfs` dependency... which requires a change in the RTD advanced settings that will affect all branches. I'm not sure how RTD reacts when we point it to a requirements file that doesnt exist... so there is a really weird dependency loop here. I'm not sure what the best answer is but just bringing this up now as a potential blocker for this PR to be pulled in.~

RTD needs to be toggled to install our pip dependency list in order to suck in the `git-lfs` package.. testing that now

![image](https://user-images.githubusercontent.com/1727935/41921006-69712802-7951-11e8-944a-05d143bcbed5.png)


## Checklist

### If you made changes to documentation:
